playbooks/adhoc/uninstall.yml

```diff
@@ -347,8 +347,7 @@
     - /etc/systemd/system/etcd_container.service
 
   - name: Remove etcd data
-    shell: rm -rf /var/lib/etcd/*
-    failed_when: false
+    file: path=/var/lib/etcd/ state=absent
```

Comment:
I think we actually used to use this, but moved away, since it actually removes /var/lib/etcd, which will cause a problem when trying to re-install.

@sdodson thoughts?